apps/builder/app/builder/features/settings-panel/shared.tsx

```diff
@@ -175,6 +175,12 @@ export const useLocalValue = <Type,>(
     }
   };
 
+  const finalSave = (value: Type) => {
```

Comment:
this doesn't sound like great addition to the api, semantics with this are messy.